project-fob/Controllers/StatsController.cs

```diff
@@ -23,39 +23,36 @@ public ActionResult Index()
             return View();
         }
         public string GetStats()
-        {
-            { //- for in the same topic, : for different topic, ; for completely different statistic
-
-                byte[] meetingIdValue;
-                bool gotvalue = false;
-                gotvalue = HttpContext.Session.TryGetValue("meetingid", out meetingIdValue);
+        { //- for in the same topic, : for different topic, ; for completely different statistic
 
-                string session = meetingIdValue.ToString();
-                //string session = Session["meetingid"].ToString();
+            bool gotvalue = false;
+            gotvalue = HttpContext.Session.TryGetValue("meetingid", out var meetingIdValue);
 
-                string MeetingIdString = System.Text.Encoding.ASCII.GetString(meetingIdValue);
-                
-                Meeting meeting = db.Meeting.Include(x => x.Stats).Single(m => m.MeetingId == MeetingIdString);
+            string session = meetingIdValue.ToString();
+            //string session = Session["meetingid"].ToString();
 
-                List<Stats> stats = meeting.Stats;
+            string MeetingIdString = System.Text.Encoding.ASCII.GetString(meetingIdValue);
 
-                String tmp = "";
+            Meeting meeting = db.Meeting.Include(x => x.Stats).Single(m => m.MeetingId == MeetingIdString);
 
-                int totalAtt = 0;
-                int totalFob = 0;
-                for (int i = 0; i < stats.Count; i++)
-                {
-                    totalAtt += stats[i].Attendeescount;
-                    totalFob += stats[i].Fobcount;
-                    tmp += stats[i].Attendeescount + "-" + stats[i].Fobcount + ":";
-                }
-                tmp += ";" + totalAtt / (double)stats.Count + "-" + totalFob / (double)stats.Count;
-                
-                //TODO use these stats (they should be in order as a list is deterministic)
+            List<Stats> stats = meeting.Stats;
 
-                return tmp;
+            String tmp = "";
```

Comment:
`tmp` is another icky name :) 

I know it's not your name, cos it's been here a while, but the [Boy Scount rule of coding](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule) is a good one to follow :)